src/v/pandaproxy/schema_registry/test/client_utils.h

```diff
@@ -117,3 +130,27 @@ inline int get_body_error_code(const ss::sstring& body) {
     }
     return obj["error_code"].Get<int>();
 }
+
+inline ppj::error_body get_error_body(const ss::sstring& body) {
+    json::Document doc;
+    if (doc.Parse(body).HasParseError()) {
+        throw ppj::parse_error(doc.GetErrorOffset());
+    }
+    if (!doc.IsObject()) {
+        throw ppj::exception_base{
+          ppj::error_code::invalid_json, "Body is not an object"};
+    }
+    auto obj = doc.GetObject();
+    if (!obj["error_code"].IsInt()) {
```

Comment:
do you have test that the key exists first?

Reply:
Done